pkgs/by-name/ec/ec2-instance-selector/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "ec2-instance-selector";
+  version = "3.1.1";
+
+  src = fetchFromGitHub {
+    owner = "aws";
+    repo = "amazon-${pname}";
```

referencing pname is bad practice, see https://github.com/NixOS/nixpkgs/issues/277994  
```suggestion
    repo = "amazon-ec2-instance-selector";
```